components/scream/scripts/scripts-tests

```diff
@@ -62,13 +62,17 @@ def run_cmd_store_output(test_obj, cmd, output_file):
 ###############################################################################
     output_file.parent.mkdir(parents=True, exist_ok=True)
     output = run_cmd_assert_result(test_obj, cmd, from_dir=TEST_DIR)
+    head = get_current_head()
+    output = output.replace(head, "CURRENT_HEAD_NORMALIZED")
```

Comment:
Curious: what is the reason for this?

Reply:
Interesting. I have always generated baselines with origin/master (I run with `-i -l` on my laptop), and never got diffs.

Reply:
@bartgol , sounds like you're talking about test-all-scream. I'm talking about scripts-tests (which calls test-all-scream and stores all the terminal output as a baseline).

Reply:
Aaaaah, ok, sorry. Got confused. Thanks!